crates/pop-cli/src/cli.rs

```diff
@@ -292,7 +292,7 @@ pub(crate) mod tests {
 			self
 		}
 
-		pub(crate) fn expect_select(
+		pub(crate) fn expect_select<T>(
```

Comment:
I don't understand why this is added again? It is not used

Reply:
My mistake, I reintroduced it because I thought it was needed for some of the new types, but you're absolutely right. It's been refactored accordingly in this commit https://github.com/r0gue-io/pop-cli/pull/316/commits/2eb51839bf8f55a3d6ea92d76ec4b9d5b297df3f.